### PR TITLE
Updates for prototype 6

### PR DESCRIPTION
**Pull request by JessicaMeixner-NOAA**
This PR turns wave initial conditions back on.  There were also several bug fixes: 
* for dddmp to get correctly to the input.nml 
* setting FISCL=9999, until bug fixes for using FISCL=0 are in the model code
* a workaround for getting threads set correctly (Issue #283)

### CODE REVIEW

File: workflow/defaults/fv3_gfs.yaml [hunk 1]
```diff
@@ -116,7 +116,6 @@ fv3_gfs_defaults: &fv3_gfs_defaults
            hord_xx_nh_nonmono: 5
            vtdm4_nh_nonmono: 0.02
            nord: 2
-           dddmp: 0.1
```
Comment (WalterKolczynski-NOAA) [inline workflow/defaults/fv3_gfs.yaml:119]: Does it not work if this is set? My understanding is these values should be getting overridden by what is in the case file, and if not I should dig in and figure it out. If the defaults are removed, workflow might fail if dddmp is *not* set.

**Review by WalterKolczynski-NOAA — CHANGES_REQUESTED** [review]
See question about dddmp defaults.

Also, in the future, please follow the [new guidelines](https://github.com/NOAA-EMC/global-workflow/wiki/Development-of-Global-Workflow#commit-message-standards) for all commit messages in branches that will be merged back to global-workflow.

File: workflow/config/fcst.yaml [hunk 2]
```diff
@@ -119,6 +118,7 @@ config_fcst:
     export dnats={doc.fv3_gfs_settings.phy_dependent_var.dnats}
     export do_sat_adj="{tools.fort(doc.fv3_gfs_settings.phy_dependent_var.do_sat_adjust)}"
     export nwat={doc.fv3_gfs_settings.phy_dependent_var.nwat}
+    export dddmp={doc.fv3_gfs_settings.dddmp}
```
Comment (JessicaMeixner-NOAA) [inline workflow/config/fcst.yaml:121]: @WalterKolczynski-NOAA ... I don't know if it was adding this line or the removal of the other lines or both that are needed for the dddmp to get set to the value that was set in the case file.  But as it was before, even though the case file said dddmp=.2, it was being set as .1

File: workflow/defaults/fv3_gfs.yaml (hunk 1 above)
Comment (JessicaMeixner-NOAA) [reply workflow/defaults/fv3_gfs.yaml:119]: Replied on another line in response to this.

**Review by WalterKolczynski-NOAA — APPROVED** [review]
Going to approve to get this in, but hopefully I'll remember to circle back to the defaults issue.

**Comment by JessicaMeixner-NOAA** [comment]
Thanks @WalterKolczynski-NOAA

**Pull request merged**